modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/components/OrderableTable.tsx

```diff
@@ -291,7 +291,7 @@ const OrderableTable = ({
 	actions,
 	className,
 	creationMenuItems,
-	creationMenuLabel = Liferay.Language.get('add'),
+	creationMenuLabel = Liferay.Language.get('new-filter'),
```

Comment:
`OrderableTable` is a general-purpose component, it should not contain filter specifics. The default here should be `New`, and filter label should be set in the `Filters` component.

Reply:
Made changes, thanks!